go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/edgexfoundry/go-mod-configuration
+
+require (
+	github.com/cenkalti/backoff v2.2.1+incompatible // indirect
+	github.com/hashicorp/consul/api v1.1.0
+	github.com/mitchellh/consulstructure v0.0.0-20190329231841-56fdc4d2da54
+	github.com/mitchellh/copystructure v1.0.0 // indirect
+	github.com/pelletier/go-toml v1.2.0
+	github.com/stretchr/testify v1.3.0
+)
```

Comment:
Should have minimum Go version declared at the end. Example

https://github.com/edgexfoundry/go-mod-core-contracts/blob/93abd0d268c8b86928b371cca9cd9e94d7b05568/go.mod#L12